pkgs/development/mobile/pebblesdk/pebble-libs.nix

```diff
@@ -0,0 +1,59 @@
+{ pkgs, python27Packages }:
+
+rec {
+  libpebble2 = python27Packages.buildPythonPackage rec {
+    name = "${pname}-${version}";
+    pname = "libpebble2";
+    version = "0.0.26";
+    src = python27Packages.fetchPypi {
+      inherit pname version;
+      sha256 = "16n69xxma7k8mhl8birdwa0fsqvf902g08s80mjb477s4dcxrvaz";
+    };
+
+    # Tries and fails to pull in pytest-mock
+    doCheck = false;
+
+    propagatedBuildInputs = with python27Packages; [ enum34 pyserial six websocket_client ];
+
+    meta = with pkgs.stdenv.lib; {
+      homepage = "";
+      license = licenses.mit;
+      description = "Library for communicating with pebbles over pebble protocol";
+    };
+  };
+
+  pypkjs = python27Packages.buildPythonPackage {
+    name = "pypkjs-1.0.6";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed. Use `pname` and `version` instead.